### ISSUE TITLE
Testing: tox

### ISSUE THREAD

**Issue by niconoe** [sckott/pygbif#67]
I noticed that it seems tox testing has been added a long time ago via: https://github.com/sckott/pygbif/pull/20, a nice addition IMHO.

Is it still in use? If no, I suggest some cleanup (`tox.ini` for example). If yes, I think we could make some improvements:

- Document tox's use in contributor's documentation
- Make sure tox environments (configured in `tox.ini` match the supported Python version - it's currently 2.7 and 3.4)
- See if can combine tox and Travic-CI: the aim would be to have a single configuration point, and allow testing all environments both locally and on Travis-CI.

Hope this helps!

**Comment by sckott** [comment]
I haven't been using tox myself. I run tests with `make test`, using nose. 

I'd be happy to try tox, can you propose changes to the file?

**Comment by niconoe** [comment]
Yeah, I'd be happy to contribute that! Don't hesitate to assign me to this issue (I can't do it myself) so there's some hope I don't forget about it :)

**Comment by sckott** [comment]
I've assigned you
